src/scripts/main.js

```diff
@@ -1,3 +1,60 @@
 'use strict';
 
-// write your code here
+// import:
+const {
+  mobileSwipes,
+} = require('./mobile');
+
+const {
+  arrowsPlayHandler,
+  createItemInRandomEmptyField,
+} = require('./move');
+
+// vars:
+const gameScore = document.querySelector('.game-score');
+const gameField = document.querySelector('.game-field');
+const allCells = gameField.querySelectorAll('.field-cell');
+const startBtn = document.querySelector('.button');
+const highestScore = document.querySelector('.highest-score');
+
+// main function:
+function gameInit() {
+  highestScore.innerText = localStorage.getItem('score') || 0;
+
+  startBtn.addEventListener('click', (e) => {
+    if (startBtn.innerText === 'Start') {
+      startBtn.innerText = 'Restart';
+      startBtn.classList.remove(`start`);
+      startBtn.classList.add(`restart`);
+
+      document.querySelector('.message-play').classList.remove('hidden');
+      document.querySelector('.message-start').classList.add('hidden');
```

Comment:
Better to create constants for this classes, 'cause now it looks like magic strings

Reply:
Also you can save result of querySelector in constant, instead of calling this method every time